usaspending_api/accounts/views/federal_accounts_v2.py

```diff
@@ -456,7 +456,7 @@ class FederalAccountsViewSet(APIView):
     def _parse_and_validate_request(self, request_dict):
         """ Validate the Request object includes the required fields """
         fy_range = [str(i) for i in range(2001, FiscalDateTime.today().year + 1)]
-        last_fy = str(SubmissionAttributes.last_certified_fy()) or str(FiscalDateTime.today().year)
+        last_fy = str(SubmissionAttributes.latest_available_fy()) or str(FiscalDateTime.today().year)
```

Comment:
Looking at how this was implemented, it will indeed get the latest, but not the latest _closed_. Will thsi change to latest _closed_ once we have that reporting dates table pulled over from broker? If so, maybe add a TODO in that function and/or create a stub for `SubmissionAttributes.latest_closed_fy()`

e.g. for the scenario where on Nov 15th ish of 2020, the FY2021 FYP01 submission window will open, and we may get FY2021 submissions. _But_ that first period of 2021 doesn't _close_ until ... Nov 30th ish. So for 2 weeks you may grab and process/display  2021 data when we shouldn't yet be.

Reply:
The load should picked **every** submission which is certified or published in Broker and not wait for a future date to load

Reply:
This isn't used in the loader, but was changed as part of this refactor. Really this just needs to be accommodated in [DEV-5217 Establish consistent account data cutoff date for API](https://federal-spending-transparency.atlassian.net/browse/DEV-5217)

Reply:
Gotcha, yes that is a separate task already captured under DEV-5217.

Reply:
I added a note to DEV-5217.